packages/dashboard/src/managers/resource-manager.ts

```diff
@@ -74,6 +77,11 @@ export default class ResourceManager {
     this.cartIds = resources.cartIds || [];
     this.loggedInDisplayLevel = resources.loggedInDisplayLevel;
     this.emergencyLots = resources.emergencyLots || [];
+    this.supportedTasks = resources.supportedTasks || [
+      { name: 'patrol', nameRemap: undefined },
+      { name: 'delivery', nameRemap: undefined },
+      { name: 'clean', nameRemap: undefined },
```

Comment:
How about instead of `name` and `nameRemap`, we use `name` and `type`. Essentially we rename "name" -> "type" and "nameRemap" -> "name". Technically, this is only a semantic change but this change better describe how "name" is treated in the code. In terms of design, it is better if task names (display name) do not have to be unique.